sdk/cognitivelanguage/azure-ai-language-conversations/swagger/README.md

```diff
@@ -29,36 +29,86 @@ clear-output-folder: true
 no-namespace-folders: true
 python: true
 title: ConversationAnalysisClient
+tag: release_2022_05_01_preview
+openapi-type: data-plane
 version-tolerant: true
 models-mode: msrest
```

Comment:
Could we please remove this `models-mode: msrest` line?
I believe we're going to GA this release without models, so we should start with this beta release.

Reply:
that means there won't be any models, right?!
i tried removing this, and all operations accept json formats input/output
i this normal?